corehq/apps/style/templates/style/includes/analytics_kissmetrics.html

```diff
@@ -26,6 +26,7 @@
      *      Default is 2000 milliseconds.
      */
     function kmqPushSafe(args, timeout) {
+        console.log("called kissmetrics with", arguments);
```

Comment:
We have console.log for GA already. Putting it for kissmetrics as well, since this makes debugging very easy.